README.md

```diff
@@ -198,15 +197,21 @@ flags:
   -history file
     	history file to use (default "~/.grol_history")
   -max-depth int
-    	Maximum interpreter depth (default 250000)
+    	Maximum interpreter depth (default 249999)
```

Comment:
I looks like  a strange number. Couldn't you keep 25k? Or maybe set it to 20k?

Reply:
it's the constant - 1 because it's adding back + 1 when setting it, and it's doing that so the tests for recursion work somewhat exactly:

https://github.com/grol-io/grol/blob/main/main_test.txtar#L170-L177

also it's 250k not 25k or 20k - we covered that in https://github.com/grol-io/grol/pull/139#discussion_r1717057870